build-flang.sh

```diff
@@ -60,13 +66,22 @@ if [ $USE_CCACHE == "1" ]; then
       -DCMAKE_CXX_COMPILER_LAUNCHER=ccache"
 fi
 
-# Build and install libpgmath
-cd runtime/libpgmath
-mkdir -p build && cd build
+if [ -n "$EXTRA_CMAKE_OPTS" ]; then
+  CMAKE_OPTIONS="$CMAKE_OPTIONS $EXTRA_CMAKE_OPTS"
+fi
+
+# Assume the script is run where it is located.
```

Comment:
Not sure about this assumption. Also this patch removes `cd` instructions. How this script should be executed? Was there a valid usage scenario agreed?

Reply:
The `cd` command on line 64 in the original version of the script had made the assumption; I just made it more explicit. The script always had to be run as `./build-flang.sh`, in the project root.

It's possible to make the script PWD-independent, but that is not in scope for this PR.